packages/ui-components/src/components/DataLibrary/DataLibrary.js

```diff
@@ -158,7 +178,7 @@ DataLibrary.defaultProps = {
 
 DataLibrary.propTypes = {
   // options: either an array of options if single data type or map of dataType -> options
-  options: PropTypes.oneOf([PropTypes.object, PropTypes.arrayOf(optionPropType)]).isRequired,
+  options: PropTypes.oneOfType([PropTypes.object, PropTypes.arrayOf(optionPropType)]).isRequired,
```

Comment:
oneOf is passing an exact value from an array, but not for type. https://jaketrent.com/post/react-oneof-vs-oneoftype